signingscript/Dockerfile

```diff
@@ -15,6 +15,7 @@ USER app
 WORKDIR /app
 
 RUN python -m venv /app \
+ && sed -i -e "s@ | base64 -d@@g" /app/docker.d/init.sh \
```

Comment:
clever :)

Reply:
its just a temporary hack. you learn this with Nix :)

Reply:
Ah, never mind my comment below, I didn't refresh before commenting :)